draft-packages/select/KaizenDraft/Select/Select.module.scss

```diff
@@ -98,6 +99,7 @@ $focus-border-color: $color-blue-500;
     border: 6px $border-solid-border-style transparent;
     border-radius: $border-solid-border-radius;
     box-shadow: $shadow-large-box-shadow;
+    z-index: 3;
```

Comment:
Why not 2 instead of jumping straight to 3? 😄

Reply:
Wanted to have some wiggle room, just in case :)